windows/tests/9.1.x/README.md

```diff
@@ -0,0 +1,83 @@
+# Tests
+
+## Running XM or XP
+
+To start XM or XP topology based Sitecore instance with a desired set of Sitecore Modules installed (PSE, SXA, JSS or Publishing Service) please use the corresponding docker-compose yml file. For example, to start Sitecore XP with SXA and Publishing Service run the following command in PowerShell:
+
+```{.ps1}
+PS> docker-compose -f docker-compose.xp.sxa.ps.yml up
+```
+
+## Running XC
+
+### Prerequisites and Considerations
+
+- XC uses the old 9.2 *AND* new 9.3 license mechanism, so set the `LICENSE_PATH` *AND* `SITECORE_LICENSE` environment variables.
```

Comment:
add 9.1 in readme.md
remove xc not included on 9.1.1
remove xc data folders in data folder

Reply:
XC data folders are removed
XC removed from comment and readme

Reply:
9.1 is added in the readme.md